awx_collection/plugins/modules/tower_job_template.py

```diff
@@ -453,9 +453,8 @@ def main():
         new_fields['inventory'] = module.resolve_name_to_id('inventories', inventory)
     if project is not None:
         if organization_id is not None:
-            project_data = module.get_one('projects', **{
+            project_data, project = module.get_one('projects', name_or_id=project, **{
```

Comment:
this doesn't look right here. It should return just one value now.

Reply:
Thanks for catching this one.